dockerfiles/Dockerfile.openvino-rhel8

```diff
@@ -0,0 +1,102 @@
+ENV ngraph_DIR=${INTEL_OPENVINO_DIR}/runtime/cmake
+ENV LD_LIBRARY_PATH=${INTEL_OPENVINO_DIR}/inference_engine/external/gna/lib:${INTEL_OPENVINO_DIR}/deployment_tools/inference_engine/external/mkltiny_lnx/lib:$INTEL_OPENVINO_DIR/deployment_tools/ngraph/lib:${INTEL_OPENVINO_DIR}/deployment_tools/inference_engine/external/omp/lib:${INTEL_OPENVINO_DIR}/runtime/3rdparty/tbb/lib/:${IE_PLUGINS_PATH}:${LD_LIBRARY_PATH}
+ENV OpenCV_DIR=${INTEL_OPENVINO_DIR}/extras/opencv/cmake
+ENV LD_LIBRARY_PATH=${INTEL_OPENVINO_DIR}/extras/opencv/lib:${LD_LIBRARY_PATH}
+ENV HDDL_INSTALL_DIR=${INTEL_OPENVINO_DIR}/runtime/3rdparty/hddl
+ENV LD_LIBRARY_PATH=${INTEL_OPENVINO_DIR}/runtime/3rdparty/hddl/lib:${LD_LIBRARY_PATH}
+ENV LD_LIBRARY_PATH=/usr/local/lib:/usr/lib:/usr/local/lib64:/usr/lib64:/lib64:${LD_LIBRARY_PATH}
+
+# Install packages 
+RUN yum install -y yum-utils autoconf automake libtool unzip udev wget zlib-devel libffi-devel openssl-devel git make gcc && \
+    yum clean packages &&  yum clean all && rm -rf /var/cache/yum && \
+# Install python 3.8
+    cd $MY_ROOT && \
+    wget https://www.python.org/ftp/python/3.8.9/Python-3.8.9.tgz && tar xvf Python-3.8.9.tgz && rm -rf Python-3.8.9.tgz && \
+    cd Python-3.8*/ && ./configure && make && make install && \
+    cd ../ &&  mkdir -p /usr/bin/Python38 && ln -s Python-3.8.9/ /usr/bin/Python38 && \
+# libusb1.0.22
+    cd /opt/ && wget https://github.com/libusb/libusb/archive/v1.0.22.zip && \
+    unzip v1.0.22.zip && rm -rf v1.0.22.zip && cd  /opt/libusb-1.0.22 && \
+# bootstrap steps
+    ./bootstrap.sh && \ 
+    ./configure --disable-udev --enable-shared && \
+    make -j4 && \
+# configure libusb1.0.22
+    cd /opt/libusb-1.0.22/libusb && \
+    /bin/mkdir -p '/usr/local/lib' && \
+    /bin/bash ../libtool   --mode=install /usr/bin/install -c   libusb-1.0.la '/usr/local/lib' && \
+    /bin/mkdir -p '/usr/local/include/libusb-1.0' && \
+    /usr/bin/install -c -m 644 libusb.h '/usr/local/include/libusb-1.0' && \
+    /bin/mkdir -p '/usr/local/lib/pkgconfig' && \
+# Install openvino
+    cd /opt/ && mkdir intel/ && cd intel && \
+    wget https://storage.openvinotoolkit.org/repositories/openvino/packages/2022.1/l_openvino_toolkit_runtime_rhel8_p_2022.1.0.643.tgz  && \
+    tar xvf l_openvino_toolkit_runtime_rhel8_p_2022.1.0.643.tgz && \
+    rm -rf l_openvino_toolkit_runtime_rhel8_p_2022.1.0.643.tgz && \
+    mv l_openvino_toolkit_runtime_rhel8_p_2022.1.0.643 openvino_2022.1.0.643 && \
+    cd ${INTEL_OPENVINO_DIR}/install_dependencies/ && ./install_openvino_dependencies.sh -y && \
+    printf "\nexport LD_LIBRARY_PATH=\${LD_LIBRARY_PATH}:/usr/local/lib\n" >> /opt/intel/openvino_2022.1.0.643/setupvars.sh && \
+    cd /opt/libusb-1.0.22 && \
+    /usr/bin/install -c -m 644 libusb-1.0.pc '/usr/local/lib/pkgconfig' && \
+    cp /opt/intel/openvino_2022.1.0.643/runtime/3rdparty/97-myriad-usbboot.rules /etc/udev/rules.d/ && \
+    ldconfig && \
+# Install GPU runtime and drivers
```

Comment:
Does this work GPU Runtime and drivers

Reply:
Though GPU support isn't added through this dockerfile, keeping this installation of GPU drivers for future use.